jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -203,6 +204,31 @@ public void testDockerDaemon_simple() throws IOException, InterruptedException,
     assertWorkingDirectory("/home", targetImage);
   }
 
+  @Test
+  public void testExecute_dockerClient() throws IOException, InterruptedException, DigestException {
+    Assume.assumeFalse(System.getProperty("os.name").startsWith("Windows"));
+    new Command(
+            "chmod", "+x", simpleTestProject.getProjectRoot().resolve("mock-docker.sh").toString())
```

Comment:
I thought git stores *nix file permissions and will have the same permissions when checked out on *nix?

Reply:
I'll give it a try.

Reply:
Hmm, if I remove this line from the gradle test, it works, but if I remove it from the maven test, it fails with "Build to Docker daemon failed, perhaps you should make sure Docker is installed and you have correct privileges to run it", even though the executable bit is set.

Reply:
OK, never mind then.